shared/libraries/websocket_streaming/src/websocket_streaming_server.cpp

```diff
@@ -31,26 +31,36 @@ void WebsocketStreamingServer::setStreamingPort(uint16_t port)
     this->streamingPort = port;
 }
 
+void WebsocketStreamingServer::setControlPort(uint16_t port)
+{
+    this->controlPort = port;
+}
+
 void WebsocketStreamingServer::start()
 {
     if (!device.assigned())
         throw InvalidStateException("Device is not set.");
     if (!context.assigned())
         throw InvalidStateException("Context is not set.");
-    if (streamingPort == 0)
+    if (streamingPort == 0 || controlPort == 0)
         return;
 
     streamingServer.onAccept([this](const daq::streaming_protocol::StreamWriterPtr& writer) { return device.getSignalsRecursive(); });
-    streamingServer.start(streamingPort);
+    // TODO implement subscribe/unsubscribe callbacks
+    streamingServer.onSubscribe([](const std::string& signalId) {} );
+    streamingServer.onUnsubscribe([](const std::string& signalId) {} );
```

Comment:
Is the actual subscription implementation then part of a different PR?

Reply:
Yes, it is part of this ticket https://blueberrydaq.atlassian.net/browse/TBBAS-903